### DISCUSSION TITLE
Option to disable Gutenberg?

### DISCUSSION THREAD

**Discussion by rwkyyy** [elementor/elementor#23380] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Mostly a seo and performance perspective, but would really like to see an option to disable WordPress Gutenberg (Blocks Builder) and not rely on a separate plugin for this. Having two pagebuilders (Blocks & Elementor) is really not optimal from a performance stand point.

### Describe the solution you'd like

An option in experimentes / features where you can disable gutenberg globally or per post type.

### Describe alternatives you've considered

installing a dedicated plugin like this one: 
[Disable Gutenberg](https://wordpress.org/plugins/disable-gutenberg/)

**Comment by hitsgee** [comment]
You can add this function to your theme function.php file if you really want to disable.

add_filter('use_block_editor_for_post', '__return_false');

**Comment by rwkyyy** [comment]
@hitsgee I want to disable the assets as well, not just the editor; the assets are mainly the problem.

**Reply by hitsgee** [reply]
There is a guide on how to do this here. https://smartwp.com/remove-gutenberg-css/

**Reply by rwkyyy** [reply]
I know how to do that... but it would be more convenient to have an option to do it, no? not change every project.

**Reply by mbaierl** [reply]
Simply add it to your child theme template and it's available in every new project you start...

**Reply by hitsgee** [reply]
The Beta has this functionality added by Elementor. Install now or wait until later this month :)

**Reply by rwkyyy** [reply]
not if you have multiple projects with different starting grounds.
